### PR TITLE
[laser_filsters_jsk_patch] add installation of launch resources

### PR DESCRIPTION
**Pull request by sktometometo**
launch file in laser_filsters_jsk_patch is not found while I am trying to execute [this demo](https://github.com/jsk-ros-pkg/jsk_demos/pull/1285) with apt packaged version.
This might be because launch files are not install without specifying which directory should be installed in CMakeLists.txt.

### CODE REVIEW

File: jsk_ros_patch/laser_filters_jsk_patch/CMakeLists.txt [hunk 1]
```diff
@@ -11,3 +11,7 @@ install(TARGETS laser_scan_filters_jsk_patch
         RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
         ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
         LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
+
+install(DIRECTORY launch/
+  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/launch
```
Comment (knorth55) [inline jsk_ros_patch/laser_filters_jsk_patch/CMakeLists.txt:16]: `DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}` ? maybe you don't need launch

Comment (sktometometo) [reply jsk_ros_patch/laser_filters_jsk_patch/CMakeLists.txt]: fixed install destination

Comment (k-okada) [reply jsk_ros_patch/laser_filters_jsk_patch/CMakeLists.txt]: is this what you want?
```
k-okada@p51s:~/catkin_ws/ws_3rdparty$ ls install/share/laser_filters_jsk_patch/
cmake  footobject_filter_example.launch  footobject_filter_example.yaml  package.xml

```

Comment (sktometometo) [reply jsk_ros_patch/laser_filters_jsk_patch/CMakeLists.txt]: Yes. `footobject_filter_example.launch` is what i want.

Comment (k-okada) [reply jsk_ros_patch/laser_filters_jsk_patch/CMakeLists.txt]:
```
install(DIRECTORY a
  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
)
install(DIRECTORY b/
  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
)
install(DIRECTORY c
  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/c
)
install(DIRECTORY d/
  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/d
)
```
goes to 
```
ws_3rdparty/install/share/laser_filters_jsk_patch/a
└── a.launch
ws_3rdparty/install/share/laser_filters_jsk_patch/b.launch
ws_3rdparty/install/share/laser_filters_jsk_patch/c
└── c
    └── c.launch
ws_3rdparty/install/share/laser_filters_jsk_patch/d
└── d.launch
1 directory, 3 files
```
so
```
install(DIRECTORY a
  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
)
install(DIRECTORY d/
  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/d
)
```
is correct.

**Pull request merged**